microshift_running_apps/microshift-deploy-with-mirror-registry.adoc

```diff
@@ -0,0 +1,25 @@
+:_content-type: ASSEMBLY
+[id="microshift-deploy-with-mirror-registry"]
+= {product-title} deployment with a mirror registry
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-deployment-mirror
+
+toc::[]
+
+You can use a custom container registry when you deploy {product-title} in an air-gapped network.
+
+include::modules/microshift-mirror-container-images.adoc[leveloffset=+1]
+
+include::modules/microshift-get-mirror-reg-container-image-list.adoc[leveloffset=+1]
+
+include::modules/microshift-mirroring-prereqs.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+* link:https://access.redhat.com/documentation/en-us/openshift_container_platform/{ocp-version}/html/installing/disconnected-installation-mirroring#installation-adding-registry-pull-secret_installing-mirroring-disconnected[Configuring credentials that allow images to be mirrored]
```

Comment:
This link returns a 404. Not sure if this is intentional or not

Reply:
Yes, it's because 4.14 isn't published yet.